fiftyone/core/aggregations.py

```diff
@@ -2884,20 +2884,30 @@ def to_mongo(self, sample_collection, big_field="values", context=None):
             context=context,
         )
 
-        self._field = self._manual_field or field
-        self._big_field = big_field
-        self._num_list_fields = len(list_fields)
+        # Optimization: call str() in memory rather than $toString in database
+        if id_to_str:
+            field = fof.ObjectIdField()
+            id_to_str = False
 
-        pipeline.extend(
-            _make_extract_values_pipeline(
+        # Optimization: use field inclusion syntax when possible
+        if self._expr is None and path == "_id":
+            big_field = path
+            _pipeline = [{"$project": {"_id": 1}}]
```

Comment:
What is big_field? I think I looked into it once, and it looked like it was just used for frames... I may be wrong, but I don't think frame _ids are used much, just sample_id, frame_number, and in that case, we should project those fields

Reply:
`values()` is a "big" aggregation because it returns IDs one at a time (in separate documents). All other aggregations are true aggregations: they do the combination in the database and return the results in a single document (ie `min()`, `max()`, etc).

Returning `values()` results as "big" style aggregations is necessary to avoid errors when the dataset contains >16MB of IDs.